src/Kunstmaan/MediaBundle/Repository/FolderRepository.php

```diff
@@ -105,10 +105,18 @@ private function deleteChildren(Folder $folder)
      */
     public function getAllFolders($limit = null)
     {
-        $qb = $this->createQueryBuilder('folder')
-            ->select('folder')
-            ->where('folder.parent is null AND folder.deleted != true')
-            ->orderBy('folder.name');
+        if ($this->_em->getConnection()->getDatabasePlatform()->getName() == 'postgresql') {
```

Comment:
Why is this check necessary?  The mysql variant can also use the new parameter, so we can avoid duplicated queries.

Reply:
Good point. In hindsight I do not know why I did that either. Perheaps being a little overzealous trying to protect backward compatibility. No matter. check will be removed.

Reply:
Absolutuetly right: ccheck is taken out